hacks/ks-verify-helm.rb

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env ruby
```

Comment:
Might be worthwhile to add a `.ruby-version` or a `Dockerfile` pointing to a supported ruby version.

Reply:
It will work with any Ruby over 2.0, but the Dockefile idea is interesting.